index.html

```diff
@@ -1,107 +1,95 @@
-    <link rel="stylesheet" href="css/style.css">
-</head>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
 
-<body id="container">
+  <body id="container">
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-<nav id="navbar"> 
-    <header> Recipes Documentation </header>
-    <ul>
-        <li> 
-            <a class="nav-link" href="#introduction" > Introduction </a>  
-        </li>    
-        <li> 
-            <a class="nav-link" href="#required_skills"> Required Skills </a>    
-        </li>    
-        <li> 
-            <a class="nav-link" href="#required_materials"> Required Materials </a>
-        </li>    
-        <li> 
-            <a class="nav-link" href="#food_recipes"> Food Recipes </a>
-        </li>    
-        <li> 
-            <a class="nav-link" href="#drink_recipes"> Drink Recipes </a>
-        </li>    
-        
-        
-    </ul>
-    
-</nav>
+    <nav id="navbar">
+      <header>Recipes Documentation</header>
+      <ul>
+        <li>
+          <a class="nav-link" href="#introduction"> Introduction </a>
+        </li>
+        <li>
+          <a class="nav-link" href="#required_skills"> Required Skills </a>
+        </li>
+        <li>
+          <a class="nav-link" href="#required_materials">
+            Required Materials
+          </a>
+        </li>
+        <li>
+          <a class="nav-link" href="#food_recipes"> Food Recipes </a>
+        </li>
+        <li>
+          <a class="nav-link" href="#drink_recipes"> Drink Recipes </a>
+        </li>
+      </ul>
+    </nav>
 
-<main id="main-doc">
-
-
-<section class="main-section" id="introduction">
-    <header>Introduction</header>
-    <article>
-        <p> Start of first article. </p> <!--Funcion de la linea de codigo-->
-        <code>
+    <main id="main-doc">
+      <section class="main-section" id="introduction">
+        <header>Introduction</header>
+        <article>
+          <p>Start of first article.</p>
+          <!--Funcion de la linea de codigo-->
+          <code>
             "function greetMe(yourName) { alert("Hello " + yourName); }
             greetMe("World");"
-        </code>
-        <p> End of first article. </p> <!--Resumen en palabras propias-->
-    </article>
-</section>
-<section class="main-section" id="required_skills">
-    <header>Required skills</header>
-    <article>
-        <p> Start of second article. </p>
-        <code>
-            "x = 42."
-        </code>
-        <subheader>List of skills.</subheader>
-        <ul>
+          </code>
+          <p>End of first article.</p>
+          <!--Resumen en palabras propias-->
+        </article>
+      </section>
+      <section class="main-section" id="required_skills">
+        <header>Required skills</header>
+        <article>
+          <p>Start of second article.</p>
+          <code> "x = 42." </code>
+          <subheader>List of skills.</subheader>
+          <ul>

```

Comment:
Please use h2-h6 to define a subheader, as the tag subheader causes a conflict when located inside the tag article. 👌